Dockerfile

```diff
@@ -505,6 +505,7 @@ RUN pip install --upgrade dask && \
 
 # Add BigQuery client proxy settings
 ENV PYTHONUSERBASE "/root/.local"
+ADD patches/kaggle.py /root/.local/lib/python3.6/site-packages/kaggle.py
```

Comment:
I'm worried that we are going to see clashes with the Kaggle API, and perhaps with the competition support injected by the worker at container creation time.  Can you make sure this is fine with @rysteboe (for the former) and @dster2 (for the latter)?

Reply:
Can't say I'm well-informed on what's going on here, but certainly we do need to keep support for the `kaggle.competitions.twosigmanews` module in Kernels, which is not installed as part of the public `kaggle` module.

Reply:
I can't find where kaggle/competitions/twosigmanews gets added to PYTHONPATH - I wonder if I can change how that gets done to so that it instead just copies competitions/ into an existing kaggle/ package folder created in this Dockerfile (which loads this kaggle.py file into it) that way everything is under the same namespace.
Alternatively maybe it is safer to just give it a different name? ie. from KaggleGCP import DataProxyClient?

Reply:
I investigated this a bunch today, and testing end to end I was able to see that indeed the namespaces collide and you lose kaggle.competitions.twosigmanews with this current approach. Finally I got this as a solution and tested it end to end: https://github.com/Kaggle/Kaggle.SDS.Worker/pull/543.